src/hooks/use-fetch.test.js

```diff
@@ -0,0 +1,10 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+import useFetch from './use-fetch';
+
+const date = '2022-04-27';
+
+describe('useFetch test', () => {
+	test("useFetch retrieves NASA's pictures", async () => {
+		await act(async () => renderHook(() => useFetch(date)));
+	});
```

Comment:
the `useFetch` returns `{ status, data }`, you should test that the hook returns the expected value and also the different states of the hook, the loading, the errors, etc.